### PR TITLE
Create update_secret_with_alias.php

### PR DESCRIPTION
**Pull request by nataliesalvati**
Update samples for Access By Alias Launch

### CODE REVIEW

**Comment by bshaffer** [comment]
This looks good but it needs a test!

**Comment by nataliesalvati** [comment]
Think I've updated the test properly.

File: secretmanager/test/secretmanagerTest.php [hunk 1]
```diff
@@ -288,3 +288,16 @@ public function testUpdateSecret()
         $this->assertStringContainsString('Updated secret', $output);
     }
 }
+public function testUpdateSecretWithAlias()
```
Comment (bshaffer) [inline secretmanager/test/secretmanagerTest.php:291]: fix CS

```suggestion

    public function testUpdateSecretWithAlias()
```

File: secretmanager/src/update_secret_with_alias.php [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+// Import the Secret Manager client library.
+use Google\Cloud\SecretManager\V1\Secret;
+use Google\Cloud\SecretManager\V1\SecretManagerServiceClient;
+use Google\Protobuf\FieldMask;
+
+/**
+ * @param string $projectId Your Google Cloud Project ID (e.g. 'my-project')
+ * @param string $secretId  Your secret ID (e.g. 'my-secret')
+ */
+function update_secret(string $projectId, string $secretId): void
```
Comment (bshaffer) [inline secretmanager/src/update_secret_with_alias.php:38]:
```suggestion
function update_secret_with_alias(string $projectId, string $secretId): void
```

**Review by bshaffer — APPROVED** [review]

**Pull request merged**